frontend/ui/screensaver.lua

```diff
@@ -299,7 +172,7 @@ local function addOverlayMessage(widget, widget_height, text)
     return widget
 end
 
-function Screensaver:chooseFolder()
+function Screensaver.chooseFolder()
```

Comment:
Isn't it a bit surprising to have a few of these with `.` even if self isn't really needed here?